doc/start/hardware-recommendations.rst

```diff
@@ -75,36 +77,38 @@ Memory
+- Setting the :confval:`osd_memory_target` below 2GB is typically not
+  recommended (it may fail to keep the memory that low and may also cause
+  extremely slow performance.
 
 - Setting the memory target between 2GB and 4GB typically works but may result
-  in degraded performance as metadata may be read from disk during IO unless the
+  in degraded performance: metadata may be read from disk during IO unless the
   active data set is relatively small.
 
-- 4GB is the current default osd_memory_target size and was set that way to try
-  and balance memory requirements and OSD performance for typical use cases.
+- 4GB is the current default :confval:`osd_memory_target` size.  This default
+  was chosen for typical use cases, and is intended to balance memory
+  requirements and OSD performance for typical use cases.
 
-- Setting the osd_memory_target higher than 4GB may improve performance when
-  there are many (small) objects or large (256GB/OSD or more) data sets being
-  processed.
+- Setting the :confval:`osd_memory_target` higher than 4GB can improve
+  performance when there many (small) objects or large (256GB/OSD or more) data
+  sets are processed.
 
-.. important:: The OSD memory autotuning is "best effort".  While the OSD may
+.. important:: OSD memory autotuning is "best effort". Although the OSD may
    unmap memory to allow the kernel to reclaim it, there is no guarantee that
-   the kernel will actually reclaim freed memory within any specific time
-   frame.  This is especially true in older versions of Ceph where transparent
-   huge pages can prevent the kernel from reclaiming memory freed from
+   the kernel will actually reclaim freed memory within a specific time
+   frame. This is especially true in older versions of Ceph where transparent
+   huge pages can prevent the kernel from reclaiming memory that was freed from
    fragmented huge pages. Modern versions of Ceph disable transparent huge
-   pages at the application level to avoid this, though that still does not
-   guarantee that the kernel will immediately reclaim unmapped memory.  The OSD
-   may still at times exceed it's memory target.  We recommend budgeting around
-   20% extra memory on your system to prevent OSDs from going OOM during
-   temporary spikes or due to any delay in reclaiming freed pages by the
-   kernel.  That value may be more or less than needed depending on the exact
-   configuration of the system.
+   pages at the application level to avoid this, but that does not
+   guarantee that the kernel will immediately reclaim unmapped memory. The OSD
+   may still at times exceed its memory target. We recommend budgeting 
+   approximately 20% extra memory on your system to prevent OSDs from going OOM
+   (**O**\ut **O**\f **M**\emory) during temporary spikes or due to delay in
+   the kernel reclaiming freed pages. That 20% value might be more or less than
+   needed, depending on the exact configuration of the system.
```

Comment:
I also have found benefit to setting `vm.min_free_kybytes` to 4MB for a similar reason, not sure if that's in-scope.